pages/en/new-design/index.mdx

```diff
@@ -1,65 +1,115 @@
+          <DownloadButton release={release}>Download Node.js (LTS)</DownloadButton>
+          <small>
+            Downloads Node.js <b>{release.versionWithPrefix}</b> with long-term support.
+            Node.js can also be installed via <a href="/download/package-manager">package managers</a>.
+          </small>
+        </>
+      )}
+    </WithNodeRelease>
+    <WithNodeRelease status="Current">
+      {({ release }) => (
+        <small>
+          Want new features sooner?
+          Get <b>Node.js <DownloadLink release={release}>{release.versionWithPrefix}</DownloadLink></b> instead.
+        </small>
       )}
     </WithNodeRelease>
-
-    <Button kind="secondary" href="/learn">Get Started</Button>
-
   </div>
 </section>
 
 <section>
-  <CodeTabs
-    languages="bash|bash"
-    displayNames="macOS / Linux (nvm)|Windows (Chocolatey)"
-    linkUrl="/download/package-manager"
-    linkText="More Options"
-  >
-    ```bash
-    # Install Node Version Manager (NVM)
-    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash
-
-    # Install Node.js
-    nvm install --lts
-
-    # Check that Node is installed
-    node -v
-
-    # Check your NPM version
-    npm -v
+  <div>
+    ```js displayName="Create an HTTP Server"
+    import { createServer } from 'node:http';
+
+    const server = createServer((req, res) => {
+      res.writeHead(200, { 'Content-Type': 'text/plain' });
+      res.end('Hello World!\n');
+    });
+
+    // starts a simple http server locally on port 3000
+    server.listen(3000, '127.0.0.1', () => {
+      console.log('Listening on 127.0.0.1:3000');
+    });
+    ```
+
+    ```js displayName="Write Tests"
+    import assert from 'node:assert';
+    import test from 'node:test';
+
+    test('that 1 is equal 1', () => {
+      assert.strictEqual(1, 1);
+    });
+
+    test('that throws as 1 is not equal 2', () => {
+      // throws an exception because 1!=2
+      assert.strictEqual(1, 2);
+    });
     ```
 
-    ```bash
-    # Install Chocolatey
-    iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+    ```js displayName="Read and Hash a File"
+    import { createHash } from 'node:crypto';
+    import { readFile } from 'node:fs/promises';
 
-    # Install Node.js
-    choco install nodejs-lts
+    const hasher = createHash('sha1');
+    const fileContent = await readFile('./package.json');
 
-    # Check that Node is installed
-    node -v
+    hasher.setEncoding('hex');
+    hasher.write(fileContent);
+    hasher.end();
 
-    # Check your NPM version
-    npm -v
+    const fileHash = hasher.read();
     ```
 
-  </CodeTabs>
-  Copy and paste this snippet to install Node.js LTS via a Package Manager
+    ```js displayName="Read Streams"
+    import { createReadStream, createWriteStream } from 'node:fs';
+
+    const res = await fetch('https://nodejs.org/dist/index.json');
+    const json = await res.json(); // yields a json object
+
```

Comment:
this doesn't seem related to the rest of the code and I'd suggest we delete - unless I am missing something about the intent of this tab or the internals of `fetch`
```suggestion
```

Reply:
No no, these are two different examples of stream. One using fetch and one using fs

Reply:
haha i find that confusing but that's just me perhaps

Reply:
It might be actually. We can revisit this in a follow-up PR!